internal/commands/user/list/command_test.go

```diff
@@ -0,0 +1,277 @@
+package list
+
+import (
+	"errors"
+	"strings"
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+	"github.com/10gen/realm-cli/internal/utils/test/mock"
+)
+
+func TestUserListSetup(t *testing.T) {
+	t.Run("Should construct a Realm client with the configured base url", func(t *testing.T) {
+		profile := mock.NewProfile(t)
+		profile.SetRealmBaseURL("http://localhost:8080")
+
+		cmd := &command{inputs: inputs{}}
+		assert.Nil(t, cmd.realmClient)
+
+		assert.Nil(t, cmd.Setup(profile, nil))
+		assert.NotNil(t, cmd.realmClient)
+	})
+}
+
+func TestUserListHandler(t *testing.T) {
+
+	const projectID = "projectID"
```

Comment:
[curious] as to why you're using `const` declarations here?  a simple `projectID, appID := "projectID", "appID"` should suffice here